uo_sys_framework/CaptureZoneModule/cfgFunctions/common/fn_CaptureZoneLoop.sqf

```diff
@@ -2,20 +2,15 @@
 #include "\x\UO_FW\addons\Main\script_macros.hpp"
 UO_FW_EXEC_CHECK(SERVER);
 
-//[_logic,_zoneName,_area,_interval,_repeatable,_capArray,_timeArray,_messagesArray,_colours,_hidden,_silent,_automessages,_ratioNeeded,_cond] passed array
-params ["","_zoneName","","_interval","","","","","","","","","",["_cond","true",[""]]];
+//[_logic,_zoneName,_area,_repeatable,_capArray,_timeArray,_messagesArray,_colours,_hidden,_silent,_automessages,_ratioNeeded,_cond] passed array
+params ["","_zoneName","","","","","","","","","","",["_cond","true",[""]]];
```

Comment:
another possibility is to also pass code {} as an argument. Advantage is that you do not need to do `call compile`